LibGAP.jl/src/gap_to_julia.jl

```diff
@@ -13,6 +13,7 @@ of converted objects and should never be given by the user.
 gap_to_julia(::Type{GAPInputType},x::GAPInputType) = x
 gap_to_julia(::Type{Any},         x::GAPInputType) = gap_to_julia(x)
 gap_to_julia(::Type{Any},         x::Any         ) = x
+gap_to_julia(::Any,               x::Nothing     ) = nothing
```

Comment:
BTW, why is this `::Any` and now `::Type{Any}` as above? I don't understand what's going on here.